FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -278,37 +278,54 @@ - (BOOL)activateFetched {
   return didActivate;
 }
 
+- (void)activateWithCompletion:(FIRRemoteConfigActivateChangeCompletion)completion {
+  [self activateWithEitherHandler:completion deprecatedHandler:nil];
+}
+
 - (void)activateWithCompletionHandler:(FIRRemoteConfigActivateCompletion)completionHandler {
+  [self activateWithEitherHandler:nil deprecatedHandler:completionHandler];
+}
+
+- (void)activateWithEitherHandler:(FIRRemoteConfigActivateChangeCompletion)completion
+                deprecatedHandler:(FIRRemoteConfigActivateCompletion)deprecatedHandler {
   __weak FIRRemoteConfig *weakSelf = self;
   void (^applyBlock)(void) = ^(void) {
     FIRRemoteConfig *strongSelf = weakSelf;
     if (!strongSelf) {
       NSError *error = [NSError errorWithDomain:FIRRemoteConfigErrorDomain
                                            code:FIRRemoteConfigErrorInternalError
                                        userInfo:@{@"ActivationFailureReason" : @"Internal Error."}];
-      if (completionHandler) {
+      if (completion) {
         dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
callback should be called in main queue, otherwise might block UI. (altho seems not related to this PR, although seems like it's using a bad practice for all following places)

Reply:
Probably should be a separate PR, right?